test/extensions/voting-rep.js

```diff
@@ -1525,6 +1530,85 @@ contract("Voting Reputation", (accounts) => {
       const pastVote = await voting.getExpenditurePastVote(slotHash);
       expect(pastVote).to.eq.BN(REQUIRED_STAKE);
     });
+
+    it("transactions that try to execute an allowed method on Reputation Voting extension are accepted by the MTX broadcaster", async function () {
+      await colony.makeExpenditure(1, UINT256_MAX, 1);
+      const expenditureId = await colony.getExpenditureCount();
+
+      const action = await encodeTxData(colony, "setExpenditureState", [1, UINT256_MAX, expenditureId, 25, [true], ["0x0"], WAD32]);
+
+      await voting.createMotion(1, UINT256_MAX, ADDRESS_ZERO, action, domain1Key, domain1Value, domain1Mask, domain1Siblings);
+      motionId = await voting.getMotionCount();
+
+      await voting.stakeMotion(motionId, 1, UINT256_MAX, YAY, REQUIRED_STAKE, user0Key, user0Value, user0Mask, user0Siblings, { from: USER0 });
+
+      await forwardTime(STAKE_PERIOD, this);
+
+      const txData = await voting.contract.methods.finalizeMotion(1).encodeABI();
+
+      const realProviderPort = process.env.SOLIDITY_COVERAGE ? 8555 : 8545;
+      const provider = new ethers.providers.JsonRpcProvider(`http://127.0.0.1:${realProviderPort}`);
+
+      const loader = new TruffleLoader({
+        contractDir: path.resolve(__dirname, "..", "..", "build", "contracts"),
+      });
+
+      // Old and new versions of ganache (which currently represents with or without coverage...)
+      // either do or don't have the hex prefix...
+      let privateKey = ganacheAccounts.private_keys[accounts[0].toLowerCase()];
+      if (privateKey.slice(0, 2) !== "0x") {
+        privateKey = `0x${privateKey}`;
+      }
+
+      const broadcaster = new MetatransactionBroadcaster({
+        privateKey,
+        loader,
+        provider,
+      });
+      await broadcaster.initialise(colonyNetwork.address);
+
+      const valid = await broadcaster.isColonyFamilyTransactionAllowed(voting.address, txData);
+      expect(valid).to.be.equal(true);
+      await broadcaster.close();
+    });
+
+    it("transactions that try to execute a forbidden method on Reputation Voting extension are rejected by the MTX broadcaster", async function () {
+      const action = await encodeTxData(colony, "makeArbitraryTransaction", [colony.address, "0x00"]);
+
+      await voting.createMotion(1, UINT256_MAX, ADDRESS_ZERO, action, domain1Key, domain1Value, domain1Mask, domain1Siblings);
+      motionId = await voting.getMotionCount();
+
+      await voting.stakeMotion(motionId, 1, UINT256_MAX, YAY, REQUIRED_STAKE, user0Key, user0Value, user0Mask, user0Siblings, { from: USER0 });
+
+      await forwardTime(STAKE_PERIOD, this);
+
+      const txData = await voting.contract.methods.finalizeMotion(motionId.toString()).encodeABI();
+
+      const realProviderPort = process.env.SOLIDITY_COVERAGE ? 8555 : 8545;
+      const provider = new ethers.providers.JsonRpcProvider(`http://127.0.0.1:${realProviderPort}`);
```

Comment:
One benefit of having all the MTX tests in a single file is that this setup can be tucked into a `before` or `beforeEach` block

Reply:
Can we put all the MTX tests in a single block so we can have a `beforeEach` for this setup?